src/sidebar/components/notebook-results.js

```diff
@@ -0,0 +1,57 @@
+import { createElement } from 'preact';
+import propTypes from 'prop-types';
+
+import useRootThread from './hooks/use-root-thread';
+import { useStoreProxy } from '../store/use-store';
+import { countVisible } from '../util/thread';
+
+/**
+ * Render count of annotations (or filtered results) visible in the notebook view
+ * There are three possible overall states:
+ * - No results (regardless of whether annotations are filtered): "No results"
+ * - Annotations are unfiltered: "X threads (Y annotations)"
+ * - Annotations are filtered: "X results [(and Y more)]"
+ */
+function NotebookResults() {
```

Comment:
This component name to me suggests that it displays the results of the current search/filter, whereas it only displays the counts. Perhaps rename to `NotebookResultCount`?

Reply:
Done.